pkg/plugins/generic/generic_plugin.go

```diff
@@ -87,12 +87,31 @@ func (p *GenericPlugin) Apply() error {
 			return nil
 		}
 	}
+
+	// Create a map with all the PFs we will need to configure
+	// we need to create it here before we access the host file system using the chroot function
+	// because the SkipConfigVf needs the mstconfig package that exist only inside the sriov-config-daemon file system
+	vfsToConfig := map[string]bool{}
```

Comment:
please, rename this variable (and parameter) to `vfsToSkip` or something like that, as its entries seem to be true when the Virtual Functions must not be configured.

Another solution may be to make `utils.SyncNodeState(...)` receive `ifSpecs []sriovnetworkv1.Interface, ifStatuses []sriovnetworkv1.InterfaceExt` and filter out the to-skip interfcaes before calling it.